tests/reports/test_reports.py

```diff
@@ -15,7 +15,16 @@ def test_reports():
             "http://perdu.com",
             "folder",
             gmtime(),
-            "WAPITI_VERSION"
+            "WAPITI_VERSION",
+            {
+                "method": "post",
+                "url": "http://testphp.vulnweb.com/login.php",
```

Comment:
Can you please add a simple check in `tests/reports/test_reports.py::test_reports` to make sure that 
 `http://testphp.vulnweb.com/login.php` appears in the report file?

Otherwise it looks good to me :+1: